linera-rpc/src/message.rs

```diff
@@ -30,6 +30,7 @@ pub enum RpcMessage {
     DownloadCertificate(Box<CryptoHash>),
     DownloadCertificates(Box<Vec<CryptoHash>>),
     BlobLastUsedBy(Box<BlobId>),
```

Comment:
Do we need the singular versions of these RPCs at all? (Maybe something to discuss for later.)

Reply:
Maybe we can remove them in a subsequent PR.